website/content/partials/components/task-aws-ecs.mdx

```diff
@@ -0,0 +1,114 @@
+## aws-ecs (task)
+
+Launch an ECS task for on-demand tasks from the Waypoint server.
+
+This will use the standard AWS environment variables and IAM Role information to
+source authentication information for AWS, using the configured task role.
+If no task role name is specified, Waypoint will create one with the required
+permissions.
+
+### Interface
+
+### Required Parameters
+
+These parameters are used in the [`use` stanza](/docs/waypoint-hcl/use) for this plugin.
+
+#### odr_image
+
+Docker image for the Waypoint On-Demand Runners.
+
+Docker image for the Waypoint On-Demand Runners. This will
+default to the server image with the name (not label) suffixed with '-odr'.".
+
+### Optional Parameters
+
+These parameters are used in the [`use` stanza](/docs/waypoint-hcl/use) for this plugin.
+
+#### cluster
+
+Cluster name to place On-Demand runner tasks in.
+
+ECS Cluster to place On-Demand runners in. This defaults to the cluster used by the waypoint server.
+
+- Type: **string**
+- **Optional**
+
+#### execution_role_name
+
+The name of the AWS IAM role to apply to the task's Execution Role.
+
+ExecutionRoleName is the name of the AWS IAM role to apply to the task's Execution Role. At this time we reuse the same Role as the waypoint server Execution Role.
+
+- Type: **string**
+- **Optional**
+
+#### log_group
+
+Cloud Watch Log Group to use for On-Demand Runners.
+
+Cloud Watch Log Group to use for On-Demand Runners. Defaults to the log group used for runners (waypoint-runner).
+
+- Type: **string**
+- **Optional**
+
+#### odr_cpu
+
+CPU to use for the On-Demand runners.
+
+Configure the CPU for the On-Demand runners. The default is 512. See https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html for valid values.
+
+- Type: **string**
+- **Optional**
+
+#### odr_memory
+
+Memory to use for the On-Demand runners.
+
+Configure the memory for the On-Demand runners. The default is 1024. See https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html for valid values.
+
+- Type: **string**
+- **Optional**
+
+#### region
+
+AWS Region to use.
+
+AWS region to use. Defaults to the region used for the Waypoint Server.
+
+- Type: **string**
+- **Optional**
+
+#### security_group_id
+
+Security Group ID to place the On-Demand Runner task in.
+
+Security Group ID to place the On-Demand Runner task in. This defaults to the security group used for the waypoint server.
```

Comment:
There's a few lower case `waypoint`s here, but I think you have to fix it on the plugin and then regenerate the docs